cookbook/templates/default/singularity-init.erb

```diff
@@ -0,0 +1,13 @@
+#!upstart
+description "Singularity Service"
+
+env PATH=/usr/local/singularity/bin:/usr/local/sbin:/usr/sbin:/sbin:/usr/lib64/qt-3.3/bin:/usr/local/bin:/bin:/usr/bin
+env MESOS_HOME=/usr/local
+env MESOS_NATIVE_LIBRARY=/usr/local/lib/libmesos.so
+env PORT=7092
```

Comment:
port should be customizable, and probably match whatever is in the Singularity config yaml